source/blog/2017-11-07-fedora-atomic-27-on-multiarch.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Fedora 27 Atomic Host Availability on Multiple Architectures
+author: sinnykumari
+date: 2017-11-07
+published: true
+comments: false
+tags: atomic, fedora, multi-arch, ppc64le, aarch64
+published: true
+---
+
+Fedora 27 release extends Atomic Host availability from x86_64 to additional architectures which includes 64 bit ARM (AArch64) and PowerPC Little Endian (ppc64le).
+Atomic Host provides immutable operating system tree (OStree) with rollback facility. To learn more, see [Project Atomic website](http://www.projectatomic.io/).
+
+# Download
+Fedora Atomic Host for aarch64 and ppc64le are available in both ISOs and cloudImages-
+[ISO](link)
+[QCOW](link)
+[Raw image](link)
```

Comment:
Let's make these bullet points by adding the `-` character to the front of each of them.